### PR TITLE
Hsb docker mongodb

### PR DESCRIPTION
**Pull request by ScottSnyder11**
Set up a Docker image/compose with MongoDB and also created a python notebook that shows some basic functionality with the mongoDB.

Please take a look and let me know if you have any questions.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,16 +1,21 @@
...
+RUN poetry config virtualenvs.create false 
+#install all dependancies via poetry.lock and pyproject.toml
+RUN poetry install
 
 COPY ./app /code/app
 
-CMD ["uvicorn", "app.main:app", "--host", "0.0.0.0", "--port", "80"]
+EXPOSE 8000
+
+CMD ["uvicorn", "app.main:app", "--host", "0.0.0.0", "--port", "8000"]
```
Comment (HotSoupRepos) [inline Dockerfile:21]: I am curious, why was the port number changed from 80 to 8000?

**Review by lindentree — APPROVED** [review]
LGTM

**Review by HotSoupRepos — APPROVED** [review]
looks good!

File: Dockerfile (hunk 1 above)
Comment (ScottSnyder11) [reply Dockerfile]: Copy and paste issue!  I have also removed exposing port 8000 and changed the port to 80

**Pull request merged**